src/corva/models/merge/merge.py

```diff
@@ -25,23 +27,23 @@ class PartialRerunMergeEvent(CorvaBaseEvent):
         run_until: run until
     """
 
-    event_type: EventType
-    partial_well_rerun_id: int
-    rerun_mode: RerunMode
-    start: int
-    end: int
+    event_type: Optional[EventType]
```

Comment:
Ideally this field should be used to discriminate between different event types, so it should not be optional
https://docs.pydantic.dev/1.10/usage/types/#nested-discriminated-unions

Reply:
I see, it is not optional below